### ISSUE TITLE
[14.0] Conflict with owl change in mail_restrict_send_user

### ISSUE THREAD

**Issue by auroraeosrose** [OCA/social#768]
some changes in the way callbacks are made in new owl versions are throwing javascript errors when creating new items

```
Uncaught (in promise) TypeError: this.thread is undefined
    _onThreadIsLoadingAttachmentsChanged mail_restrict_send_button/static/src/js/chatter.js:175
    _flushUpdateCycle mail/static/src/model/model_manager.js:669
    update mail/static/src/model/model_manager.js:253
    update mail/static/src/models/model/model.js:208
    _onThreadCacheChanged mail/static/src/models/thread_view/thread_view.js:165
    _flushUpdateCycle mail/static/src/model/model_manager.js:669
    delete mail/static/src/model/model_manager.js:135
    delete mail/static/src/models/model/model.js:190
    _onThreadIdOrThreadModelChanged mail_restrict_send_button/static/src/js/chatter.js:111
    _flushUpdateCycle mail/static/src/model/model_manager.js:669
    update mail/static/src/model/model_manager.js:253
    update mail/static/src/models/model/model.js:208
    _insertFromProps mail_restrict_send_button/static/src/js/chatter.js:372
```

Changing the line https://github.com/OCA/social/blob/14.0/mail_restrict_send_button/static/src/js/chatter.js#L175

to also check for thread existing

```
 if (!this.thread || !this.thread.isLoadingAttachments) {
```
solves the issue

**Comment by pedrobaeza** [comment]
Can you propose the PR?

**Comment by dreispt** [comment]
Fixed by #769
